changelog/db.changelog-1.1.xml

```diff
@@ -8,10 +8,10 @@
     </changeSet>
     <changeSet author="Defra" id="4">
         <createTable tableName="commsEvent">
-            <column name="id" type="VARCHAR(36)">
+            <column name="id" type="UUID">
                 <constraints nullable="false" primaryKey="true" primaryKeyName="commsEvent_pkey" />
             </column>
-            <column name="dateCreated" type="TIMESTAMP">
+            <column name="dateCreated" type="TIMESTAMPTZ">
```

Comment:
Any reason for why we don't want to just stick with UTC timestamps?

If we want timezones we should make sure all our services are doing this.

Reply:
Adding this as an agenda item to tech huddle